### DISCUSSION TITLE
What is your preferred method of mounting network storage?

### DISCUSSION THREAD

**Discussion by JohnGuan** [IceWhaleTech/CasaOS#435] [Polls]
We are developing a way to mount network storage. 
We will adjust the development priority based on the poll results.

**Comment by Lauren-ED209** [comment]
Shall we support ...multiple selection?

**Reply by JohnGuan** [reply]
Aha! I tried that, but the poll in GitHub Discussion seems to be a single choice. So the word I used was preferred.

If anyone knows how to set up multiple choice please let me know XD
